book/src/offline-signing/durable-nonce.md

```diff
@@ -0,0 +1,201 @@
+# Durable Transaction Nonces
+
+Durable transaction nonces are a mechanism for getting around the typical
+short lifetime of a transaction's [`recent_blockhash`](../transaction.md#recent-blockhash).
+They are implemented as a Solana Program, the mechanics of which can be read
+about in the [proposal](../implemented-proposals/durable-tx-nonces.md).
+
+## Known Issues
+
+### Fee Theft Opportunity
+
+The durable nonce implementation contains a vulernability which allows for fees
+to be stolen by a transaction using the feature under certain conditions. To
+achieve this, the transaction must have failed with an instruction error, which
```

Comment:
Suggested rewrite for "To achieve this..." sentence:
"If the transaction fails with an instruction error, the runtime rolls back step that advanced the stored nonce, allowing the transaction to be replayed and fees charged again."